### ISSUE TITLE
Replies don't get highlighted

### ISSUE THREAD

**Issue by mmattbtw** [Chatterino/chatterino2#3271]
**Describe your issue**

When a user replies to a message on Twitch, on Chatterino it @ mentions you, however it doesn't actually highlight the message for that user.

**Screenshots**

<img width="407" alt="Chatterino Screenshot" src="https://user-images.githubusercontent.com/30363562/135733323-458553fd-106b-4c71-9d20-a6b02cbdbdf5.png">
<img width="581" alt="Web Chat Screenshot" src="https://user-images.githubusercontent.com/30363562/135733324-ad2f0537-8bd6-4442-aa47-ef16a79243c4.png">

**OS and Chatterino Version**

macOS Monterey v12.0 (Beta)
Chatterino 2.3.4 (commit 0ba7c0f3c)

**Comment by Mm2PL** [comment]
Unable to reproduce. Do you have pings for your username set?

Chatterino 2 9b9fd7d4

**Comment by mmattbtw** [comment]
> Unable to reproduce. Do you have pings for your username set?
> 
> Chatterino 2 [9b9fd7d](https://github.com/Chatterino/chatterino2/commit/9b9fd7d403a0b3bd047ba7134de158c4e2fecbc7)

![image](https://user-images.githubusercontent.com/30363562/135733708-2c969cb0-9d8e-44fc-a74f-406bd95ac49e.png)
Yep

**Comment by Felanbird** [comment]
Issue was resolved in chat.

**Comment by seelentau** [comment]
Hi, I'm very sorry for necroing this thread, but I'm very unfamiliar with Chatterino (and Github, for that matter lol). I have the same issue as the OP. These are my highlights:

![Image](https://github.com/user-attachments/assets/68af545a-c49e-485a-9b7d-4f1d72f193ec)

This is how regular Twitch chat looks:

![Image](https://github.com/user-attachments/assets/1ab0965c-0e4e-4b17-9aab-9a003e57f6c1)

And this is Chatterino:

![Image](https://github.com/user-attachments/assets/89843087-595e-41ad-95d3-6b9fd7919fc5)

The message does get highlighted when I turn on "Automatically subscribe to participated reply threads":

![Image](https://github.com/user-attachments/assets/454217d0-df6d-49de-8475-b9c9d579b738)

However, I would like to have this turned off, because turning it on means that I get a highlight for each and every message in each and every message thread I participated in (which are a lot, as a mod).

Is there any way to find a middle ground? I'd like to be highlighted when people reply to me without mentioning me, but not when people reply to someone else in a thread I participated in.

I'm certain there's probably a thread on this issue somewhere else, I apologize in advance. But again, I really don't know the specifics on what to search for >< I also looked through all settings, asked other more experienced users and made sure I have installed the correct and latest Chatterino version.

Kind regards
seelentau
